ReactWindows/ReactNative/UIManager/BaseViewManager.cs

```
@@ -105,6 +106,32 @@ public void SetZIndex(TFrameworkElement view, int zIndex)
         }
 
         /// <summary>
+        /// Sets the manipulation mode for the view.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="manipulationModes">The manipulation modes.</param>
+        [ReactProp("manipulationModes")]
+        public void SetManipulationMode(TFrameworkElement view, JArray manipulationModes)
+        {
+            if (manipulationModes == null || manipulationModes.Count == 0)
+            {
+                view.ManipulationMode = ManipulationModes.System;
+            }
+
+            var manipulationMode = ManipulationModes.System;
+            foreach (var modeString in manipulationModes)
+            {
+                if (modeString.Type == JTokenType.String)
```

Do we need to throw for invalid input? Should we just use Debug.Assert here?